Vagrantfile

```diff
@@ -10,7 +10,7 @@ Vagrant.configure("2") do |config|
   config.vm.synced_folder "~/.hab/cache/keys", "/hab/cache/keys"
   config.vm.synced_folder "~/.hab/etc", "/hab/etc"
 
-  config.vm.network "forwarded_port", guest: 80, host: 9636
```

Comment:
This is a shim for builder-api-proxy. Some services on the outside expect to hit builder-api on port 80. This should stay.

Reply:
If it stays, it gets overridden by the later port mapping.

Given:

```
config.vm.network "forwarded_port", guest: 80, host: 9636
config.vm.network "forwarded_port", guest: 9631, host: 9631
config.vm.network "forwarded_port", guest: 9636, host: 9636
```

The resultant mapped ports are:

| Name     | Protocol | Host IP    | Host Port | Guest IP | Guest Port |
| -------- | --------- | --------- | ---------- | -------- | ----------- |
| ssh         | TCP         | 127.0.0.1 | 2222         |                | 22               |
| tcp9631 | TCP         |                 | 9631         |                | 9631           |
| tcp9636 | TCP        |                 | 9636         |                | 9636           |